controllers/pravega_segmentstore.go

```diff
@@ -641,6 +641,7 @@ func MakeSegmentStoreExternalServices(p *api.PravegaCluster) []*corev1.Service {
 		}
 		if p.Spec.Pravega.SegmentStoreLoadBalancerIP != "" {
 			service.Spec.Ports[0].Port = int32(serviceport) + i
+			service.Spec.Ports[1].Port = int32(adminPort) + i
```

Comment:
@anishakj is it expected to go up (9999, 1000, 1001) or down (9999, 9998, 9997)?

Reply:
ports are supported upto `65535`